src/standard/configure.html

```diff
@@ -45,11 +45,13 @@
 
     // storage for configuration
     _setupConfigure: function(initialConfig) {
-      this._config = {};
+      // If we have more non-undefined properties, it is more efficient to clone whole object
+      // and then remove undesired properties
+      this._config = Object.create(initialConfig || {});
       // don't accept undefined values in intialConfig
       for (var i in initialConfig) {
-        if (initialConfig[i] !== undefined) {
-          this._config[i] = initialConfig[i];    
+        if (initialConfig[i] === undefined) {
+          delete this._config[i];
```

Comment:
This won't actually work as you expect. `delete` only deletes properties off of the given object, and does not walk the prototype chain.

```js
var a = {a: 1};
var b = Object.create(a);
delete b.a;
b.a === 1;
```

Therefore, undefined properties will still exist in `this._config`. Instead, you'd have to do something like this:
```js
this._config = initalConfig;
for (var i in initialConfig) {
  if(initialConfig[i] === undefined) {
   delete this._config[i];
 }
}
```

Also, the use of `delete` slows down property accesses in v8 by deoptimizing the object. To accept this PR, I we need a benchmark showing that using `delete` is faster than cloning the object without `undefined` values.